bazel/external/requirements.txt

```diff
@@ -4,12 +4,80 @@
 #
 #    pip-compile --allow-unsafe --generate-hashes --output-file=requirements.txt requirements.in
 #
+attrs==21.4.0 \
```

Comment:
let's add this file here so we don't have to review it in code review: https://github.com/magma/magma/blob/master/.gitattributes